### PR TITLE
CLI solver option

### PR DESCRIPTION
**Pull request by leonardoalt**
Fixes https://github.com/leonardoalt/yools/issues/17

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -55,7 +64,10 @@ fn symbolic_revert(sub_matches: &ArgMatches) -> Result<(), String> {
 
     let query = yools::encoder::encode_revert_unreachable::<EVMInstructions>(&ast);
 
-    match solver::query_smt(&query) {
+    match solver::query_smt_with_solver(
+        &query,
+        solver::SolverConfig::new(sub_matches.value_of("solver").unwrap_or("cvc4")),
```
Comment (chriseth) [inline src/main.rs:69]: Shouldn't we rather use `default_value` when setting up the options?

Comment (leonardoalt) [reply src/main.rs:69]: Whats that?

Comment (leonardoalt) [reply src/main.rs:69]: ah you mean directly in the cli options

Comment (leonardoalt) [reply src/main.rs]: pushed it

**Review by chriseth — APPROVED** [review]

**Pull request merged**